bin/spice/pkg/spec/spicepod_test.go

```diff
@@ -204,8 +204,8 @@ nested_unknown:
 	result = convertMap(result).(map[string]interface{})
 
 	// Check known fields
-	if result["version"] != "v1beta1" {
-		t.Errorf("Expected version v1beta1, got %v", result["version"])
+	if result["version"] != "v1" {
```

Comment:
same here

Reply:
it does, this is for the test where we write v1